src/seedsigner/gui/screens/settings_screens.py

```diff
@@ -296,3 +296,26 @@ def __post_init__(self):
             supersampling_factor=1,
             screen_y=self.components[-1].screen_y + self.components[-1].height + GUIConstants.COMPONENT_PADDING
         ))
+
+
+
+@dataclass
+class FirmwareScreen(BaseTopNavScreen):
+    def __post_init__(self):
+        self.title = "Firmware"
+        super().__post_init__()
+
+        self.components.append(TextArea(
+            text='''Version 0.6.0
```

Comment:
Version is already defined for the launch screen. Shouldn't hard-code it again here.

see: https://github.com/SeedSigner/seedsigner/blob/dbb1d6b1d77a06ab2b18fa3258301365e83be568/src/seedsigner/controller.py#L53